felix/bpf/proxy/kube-proxy.go

```diff
@@ -60,12 +61,13 @@ type KubeProxy struct {
 }
 
 // StartKubeProxy start a new kube-proxy if there was no error
-func StartKubeProxy(k8s kubernetes.Interface, hostname string,
+func StartKubeProxy(k8s kubernetes.Interface, hostname string, nodeLabels map[string]string,
```

Comment:
if you added the zone just as an option if the zone was available, you would have saved yourself a lot of editing in all the tests ;-)

Reply:
Thanks @tomastigera  - as suggested added the zone just an an option and revert the previous edit contract changes as a result